### ISSUE TITLE
spaceAroundParens wants to remove space between capture list and params in closure

### ISSUE THREAD

**Issue by CraigSiemens** [nicklockwood/SwiftFormat#950]
It's incorrectly giving the following warning for the following code. Removing `(value: 0)` or `<Foo>` from the code causes the warning to no longer be produced.

```
warning: (spaceAroundParens) Add or remove space around parentheses.
```

```swift
Something<Foo>(value: 0) { [weak self] (_) in
    print("hello")
}
```

is changed to
```swift
Something<Foo>(value: 0) { [weak self](_) in
    print("hello")
}
```

Config
```
--swiftversion 5.4
--rules spaceAroundParens
--rules wrapArguments
```

**Comment by nicklockwood** [comment]
@CraigSiemens thanks for reporting this, I'll get it fixed asap.

**Comment by nicklockwood** [comment]
@CraigSiemens in the meantime, a workaround would be to manually remove the `( )` around `_`, or to add:

```swift
// swiftformat:disable:next spaceAroundParens
```

on the line above the closure declaration.

**Comment by CraigSiemens** [comment]
Thanks for the suggestion and looking into it.

I actually decided to stick with the incorrect formatting so that when there's a new SwiftLint version, it'll be flagged/corrected. AFAIK the comment won't be flagged if it has no affect so we'll probably forget to remove it.

**Comment by nicklockwood** [comment]
> AFAIK the comment won't be flagged

Yes that's true (and probably something I should fix)

**Comment by nicklockwood** [comment]
@CraigSiemens fixed in 0.48.5

**Comment by CraigSiemens** [comment]
Thanks @nicklockwood!
